src/core/server/handler/handler.cpp

```diff
@@ -0,0 +1,172 @@
+#include <handler/handler.h>
+
+handler::handler(const utility::string_t &url) : m_listener(url) {
+	m_listener.support(
+		methods::GET,
+		[this](auto &&PH1) {
+		  handle_get(std::forward<decltype(PH1)>(PH1));
+		});
+	m_listener.support(
+		methods::PUT,
+		[this](auto &&PH1) {
+		  handle_put(std::forward<decltype(PH1)>(PH1));
+		});
+	m_listener.support(
+		methods::POST,
+		[this](auto &&PH1) {
+		  handle_post(std::forward<decltype(PH1)>(PH1));
+		});
+	m_listener.support(
+		methods::DEL,
+		[this](auto &&PH1) {
+		  handle_delete(std::forward<decltype(PH1)>(PH1));
+		});
+
+}
+
+[[maybe_unused]] void handler::handle_error(const pplx::task<void> &task) {
+	try {
+		task.get();
+	}
+	catch (...) {
+		// TODO: Log the error
+	}
+}
+
+// TODO: remove this
+void display_json(
```

Comment:
Maybe you have forgotten to remove this?

Reply:
It will be removed in the next PR